src/main/resources/assets/gregtech/lang/ru_ru.lang

```diff
@@ -1712,6 +1701,8 @@ tile.machine_casing.steel_hull.name=Стальной корпус
 tile.machine_casing.steel_hull.tooltip=Для улучшения паровой машины
 tile.machine_casing.steel_bricks_hull.name=Кирпичный стальной корпус
 tile.machine_casing.steel_bricks_hull.tooltip=Для улучшения паровой машины
+tile.metal_casing.primitive_bricks.name=Примитивные кирпич
```

Comment:
Do you really know russian and chinese? Russian translation makes no sense, and i guess it's the same for chinese.

Reply:
I just move the translation It's not a creation